com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -229,27 +265,54 @@ def __init__(self, json_view, rest_client):
         self._tuple_fn = tuple_fn
 
     def get_domain(self):
+        """Get the Streams domain that owns this view element resource
+
+        Returns:
+            :py:class:`Domain`: Domain owning this view element resource
+        """
         return Domain(self.rest_client.make_request(self.domain), self.rest_client)
 
     def get_instance(self):
+        """Get the Streams instance that owns this view element resource
+
+        Returns:
+            :py:class:`Instance`: Instance owning this view element resource
+        """
         return Instance(self.rest_client.make_request(self.instance), self.rest_client)
 
     def get_job(self):
+        """Get the Streams job that owns this view element resource
+
+        Returns:
+            :py:class:`Job`: Job owning this view element resource
+        """
         return Job(self.rest_client.make_request(self.job), self.rest_client)
 
     def stop_data_fetch(self):
+        """Stops the thread that fetches data from the Streams view server
+        """
         if self._data_fetcher is not None:
             self._data_fetcher.stop.set()
             self._data_fetcher = None
 
     def start_data_fetch(self):
+        """Start a thread that fetches data from the Streams view server
+
+        Returns:
+            :py:class:`queue.Queue`: Queue containing view data
+        """
         self.stop_data_fetch()
         self._data_fetcher = _ViewDataFetcher(self, self._tuple_fn)
         t = threading.Thread(target=self._data_fetcher)
         t.start()
         return self._data_fetcher.items
 
     def get_view_items(self):
+        """Get a list of ViewItem resources associated with this view resource
```

Comment:
I don't think we need 'resource' everywhere. At this point it's a ViewItem and a View.